libs/designsystem/src/lib/components/action-sheet/action-sheet-popout/action-sheet-popout.component.scss

```
@@ -33,9 +33,15 @@ kirby-card {
   button[kirby-button] {
     margin: 0;
     border-radius: 0;
+    transition: background-color get-transition-duration('quick') get-transition-easing('linear'),
+      opacity get-transition-duration('quick') get-transition-easing('linear');
```

Consider simplifying:
```suggestion
  transition-property: background-color, opacity;
  transition-duration: get-transition-duration('quick');
  transition-timing-function: get-transition-easing('linear');
```